src/Surfnet/StepupRa/RaBundle/Resources/public/less/style.less

```diff
@@ -233,9 +233,22 @@ form[name="select_institution"] {
     div.row.horizontal {
         div.form-group {
             float: left;
+            width: 60%;
+
+            .control-label {
+                text-align: left;
+            }
         }
     }
 }
+
+#select_institution_select_and_apply {
+    display: block;
+}
+.js #select_institution_select_and_apply {
```

Comment:
Nice touch to keep support for non-js believers!

Reply:
And alson the most easiest solution.